src/examples/data.cljc

```diff
@@ -953,3 +953,56 @@
    ::exception-data     exception-data
    ::http-requests      http-requests
    ::http-responses     http-responses})
+
+(def proc-par-simple
+  ^{:portal.viewer/default :portal.viewer/proc-par}
```

Comment:
FYI The metadata here gets applied to the `(list )` form and not the result list value. I think you want with-meta here.